src/main/webapp/recipe-display.html

```diff
@@ -135,9 +136,11 @@
           })
       }
 
+      /* Deletes the currently displayed post and all associated comments.
+       *   TODO: Only allow this operation to be taken by post creator
```

Comment:
This is taken care of on the back end, but yeah ideally we'll have the delete/edit posts and edit comment html elements visible only to the creator